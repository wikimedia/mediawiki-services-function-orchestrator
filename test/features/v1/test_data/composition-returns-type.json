{
	"Z1K1": "Z7",
	"Z7K1": {
		"Z1K1": "Z8",
		"Z8K1": [],
		"Z8K2": "Z1",
		"Z8K3": [],
		"Z8K4": [
			{
				"Z1K1": "Z14",
				"Z14K1": "Z10121",
				"Z14K2": {
					"Z1K1": "Z7",
					"Z7K1": "Z801",
					"Z801K1": null
				}
			}
		],
		"Z8K5": "Z10121"
	}
}
