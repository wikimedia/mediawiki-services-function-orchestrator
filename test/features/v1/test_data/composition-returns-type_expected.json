{
	"Z1K1": "Z4",
	"Z4K1": {
		"Z1K1": "Z7",
		"Z7K1": "Z10122",
		"Z10122K1": ""
	},
	"Z4K2": [
		"Z3",
		{
			"Z1K1": "Z3",
			"Z3K1": "Z6",
			"Z3K2": "K1",
			"Z3K3": {
				"Z1K1": "Z12",
				"Z12K1": [
					"Z11"
				]
			}
		}
	],
	"Z4K3": "Z101"
}
